processors/pubmed_unregistered_trials/processor.py

```diff
@@ -6,43 +6,44 @@
 
 import json
 import logging
-from sqlalchemy import exc as sqlalchemy_exceptions
 from .. import base
+from .. import pubmed_publications
 from . import extractors as extractors_module
-from ..pubmed_publications import extractors as related_extractors_module
 logger = logging.getLogger(__name__)
 
 
 def process(conf, conn):
     extractors = base.helpers.get_variables(
         extractors_module, lambda x: x.startswith('extract_')
     )
-    related_extractors = base.helpers.get_variables(
-        related_extractors_module, lambda x: x.startswith('extract_')
+    pubmed_publications_extractors = base.helpers.get_variables(
+        pubmed_publications.extractors, lambda x: x.startswith('extract_')
     )
-    source = related_extractors['extract_source'](None)
+    source = pubmed_publications_extractors['extract_source'](None)
     source_id = base.writers.write_source(conn, source)
 
     for record in base.helpers.iter_rows(conn, 'warehouse', 'pubmed', orderby='meta_id'):
-        publication = related_extractors['extract_publication'](record)
-        unregistered_trial = extractors['extract_trial'](record)
+        publication = pubmed_publications_extractors['extract_publication'](record)
+        potential_trial = extractors['extract_trial'](record)
         conn['database'].begin()
         try:
 
             # Create/Update unregistered trial from the publication
             if not publication['registry_ids']:
                 trial_id, is_primary = base.writers.write_trial(
-                    conn, unregistered_trial, source_id, record['meta_id']
+                    conn, potential_trial, source_id, record['meta_id']
                 )
                 base.writers.write_record(
-                    conn, record, source_id, trial_id, unregistered_trial, is_primary
+                    conn, record, source_id, trial_id, potential_trial, is_primary
                 )
 
             # Publication is not/no longer an unregistered trial but a results document
             else:
-                delete_record = "DELETE FROM records WHERE identifiers @> '%s'"
-                delete_record = delete_record % json.dumps(unregistered_trial['identifiers'])
-                conn['database'].query(delete_record)
+                delete_record_query = 'DELETE FROM records WHERE identifiers @> :identifiers'
+                conn['database'].query(
+                    delete_record_query,
+                    identifiers=json.dumps(potential_trial['identifiers'])
```

Comment:
@vitorbaptista I had to use `json.dumps` because passing the dict directly raises
 ` ProgrammingError: (psycopg2.ProgrammingError) can't adapt type 'dict' [SQL: 'DELETE FROM records WHERE identifiers @> %(identifiers)s'] [parameters: {'identifiers': {u'pubmed': u'PUBMED23571166'}}]
`